admin/src/components/CKEditor/index.js

```diff
@@ -73,12 +73,25 @@ const Editor = ({ onChange, name, value, disabled }) => {
       const plugin = await request(`/${pluginId}/config/plugin`, { method: "GET" });
       const upload = await request(`/${pluginId}/config/uploadcfg`, { method: "GET" });
 
+      //read i18n locale
+      const urlSearchParams = new URLSearchParams(window.location.search);
+      const params = Object.fromEntries(urlSearchParams.entries());
+      const languageContent = params["plugins[i18n][locale]"];
+
+      //set locale language code to content
+      let language = editor.language;
+      if (languageContent) {
+        const countryCode = languageContent.split("-")[0]
+        if (countryCode)
+          language = { content: editor.language.content ? editor.language.content : countryCode, ui: editor.language.ui, textPartLanguage: editor.textPartLanguage }
+      }
```

Comment:
@zadinvit Hi, thank you for your pr.
I think we need some changes here:
```js
          let language = editor.language;
          if (languageContent) {
            const countryCode = languageContent.split("-")[0]
            if (countryCode && language)
              language = { 
                content: language.content || countryCode,
                ui: typeof language === "string" && language || language.ui || auth.getUserInfo().preferedLanguage,
                textPartLanguage: language.textPartLanguage 
            }
          }
```
And this should be inside if(editor) statement below

Reply:
Hi, 
I agree with you. I move your changes to my code at my branch tomorow. Thanks for improve my code.